scraper.rb

```diff
@@ -2,12 +2,15 @@
 # encoding: utf-8
 
 require 'wikidata/fetcher'
+require 'pry'
```

Comment:
Did you really mean to introduce this, or is this left over from local development/debugging? 

I don't really mind it being here, as it's always useful to have available, but if it's deliberate it should really be introduced in a separate commit.